docs/self-managed/identity/user-guide/managing-user-access.md

```diff
@@ -0,0 +1,45 @@
+---
+id: managing-user-access
+title: "Managing user access to Identity"
+sidebar_label: "Managing user access"
+---
+
+In this guide, you will learn how to grant users different levels of access to the Identity application.
+
+## Permissions supported by Identity
+
+Identity implements the following permissions:
+
+- `read`: Users can access all pages in Identity. They _cannot_ create, modify, or delete any data.
+- `read:users`: Users can access only the **Users** page and related subpages.
+- `write`: Users have access to all pages. They can create, modify, and delete data.
+
+:::note
+If you are connecting Identity to your existing Keycloak installation, the `read:users` permission needs to be created
+manually in Keycloak.
```

Comment:
Do we have a guide on how to do this? In theory with 8.1 hopefully it won't be an issue as we will [adapt Identity to handle this for the user](https://github.com/camunda-cloud/identity/issues/983) but I can see users getting stuck with this task as we require the permissions to be configured in a specific way.

As we have a planned piece of work to negate this issue I'd be tempted to remove this note for now as its easier than adding in a temporary page on adding permissions in Keycloak :)

Reply:
Do you think it would be enough to link to the [Keycloak docs on managing permissions](https://www.keycloak.org/docs/16.1/authorization_services/index.html#_permission_overview)? I can imagine that we would at least need some basic explanations on which options to choose in Keycloak, so I would be fine removing the note.

Reply:
Linking to that page would definitely work though the more I think about it I would personally opt for removing the note for now, we can always add something in if we don't manage to remove the requirement for the user to manually create the permission

Reply:
That sounds reasonable 👍 I removed the note for now.